code/_onclick/item_attack.dm

```diff
@@ -45,18 +45,21 @@
 	user.changeNext_move(CLICK_CD_MELEE)
 	return I.attack(src, user)
 
-
 /obj/item/proc/attack(mob/living/M, mob/living/user)
 	if(SEND_SIGNAL(src, COMSIG_ITEM_ATTACK, M, user) & COMPONENT_ITEM_NO_ATTACK)
 		return
 	SEND_SIGNAL(user, COMSIG_MOB_ITEM_ATTACK, M, user)
 	if(item_flags & NOBLUDGEON)
 		return
-
-	if(force && HAS_TRAIT(user, TRAIT_PACIFISM))
-		to_chat(user, "<span class='warning'>You don't want to harm other living beings!</span>")
-		return
-
+	if(force)
+		if(HAS_TRAIT(user, TRAIT_PACIFISM))
+			to_chat(user, "<span class='warning'>You don't want to harm other living beings!</span>")
+			return
+		if(istype(user, /mob/living/carbon/human))

```

Comment:
```suggestion
		if(HAS_TRAIT(user, MONK_TRAIT))
			to_chat(user, "<span class='warning'>Your vows as a Monk prevent you from using [src] in combat!</span>")
			return
	
```

Reply:
Thanks, when I have time I'll make the suggested change.

Reply:
Tested out the code and unfortunately it doesn't work.